### ISSUE TITLE
Support setuptools entry points for unit2 scripts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#12]
```
From Chris Withers:

http://bitbucket.org/chrisw/unittest2/changeset/6689473d4571

```

Original issue reported on code.google.com by `fuzzyman` on 20 Apr 2010 at 6:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `fuzzyman` on 20 Apr 2010 at 6:50
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.4.2.
```

Original comment by `fuzzyman` on 6 Jun 2010 at 7:06
- Changed state: **Fixed**
